.bacon.yml

```diff
@@ -9,7 +9,7 @@ test_suites:
     - name: verify-package
       script_path: /root/okta/okta-signin-widget/scripts
       sort_order: '2'
-      timeout: '10'
+      timeout: '15'
```

Comment:
why would this script need more time?

Reply:
I think it's because all new dependencies and yarn build (prepare) time increase

Reply:
@lesterchoi-okta maybe also increase timeout for `publish`, I remember last build almost hit the timeout limit